src/views/user/utils/FormField.vue

```diff
@@ -40,6 +40,14 @@
         class="input"
         :class="{ 'is-danger': hasError || errorMessage }"
       />
+      <span
+        v-if="type === 'password'"
+        class="icon is-small is-right"
+        @click="onMaskToggle"
+        style="pointer-events: initial"
+      >
```

Comment:
Please use a class instead, e.g. 'is-clickable' helper from bulma css.

```suggestion
      <span
        v-if="type === 'password'"
        class="icon is-small is-right is-clickable"
        @click="onMaskToggle"
      >
```